src/target.rs

```diff
@@ -13,7 +14,9 @@ impl FromStr for Target {
     type Err = ();
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        //"" => Ok(Target::NATIONAL)
+        if s.is_empty() {
```

Comment:
I'd call `s.trim().is_empty()` to be sure (I was playing with this branch and some of the clap tweaks mentioned and I notice without the `trim()` sometimes the input could be erroneously be parsed as a postcode with a blank string)